hasktorch/src/Torch/Typed/Functional.hs

```diff
@@ -454,6 +454,8 @@ type family SqueezeAll (shape :: [Nat]) :: [Nat] where
   SqueezeAll (x: xs) = x ': SqueezeAll xs
 
 -- | squeezeAll
+-- | Note: this function is unsafe; dimensions not known statically are retained in the type,
+-- | but may be squeezed out if they turn out 1 at run-time.
```

Comment:
Note is good.
Could you add the document of behavior?

Reply:
@junjihashimoto wait I thought this was documenting behavior, I think I'm confused about what you meant. :sweat_smile: